File: SportsStore.Domain/Entities/Order.cs

using System.Collections.Generic;
namespace SportsStore.Domain.Entities
{
    public class Order
    {
        public int OrderId { get; set; }

        public int CustomerId { get; set; }

        public List<OrderDetail> OrderDetails { get; set; }

        public decimal Total { get; set; }
    }
}

File: SportsStore/SportsStore/SportsStore.WebUI/Scripts/models/Order.ts
module app {
    'use strict'
    export class Order {
        orderId: number;
        orderNo: string;
        customerId: number;
        customerName: number;
        email: string;
        address: string;
        phone: string;
        comment: string;
        orderDetails: OrderDetail[];
        total: number;
        orderDate: Date;
        status: number;

        constructor() {
        }
    }
}